Entra conditional access: Migration (#32389)
fleetdm/fleet: articles/entra-conditional-access-integration.md

When a device fails a Fleet policy, Fleet can mark it as non-compliant in Entra. This allows IT and Security teams to block access to third-party apps until the issue is resolved.

> Migrating from your current MDM to Fleet? Head to the [migration instructions](#migration).

Author: ```suggestion
Migrating from your current MDM solution to Fleet? Head to the [migration instructions](#migration).
```

If a user turns off MDM by going to **System Settings > Device Management and selecting **Unenroll** on Fleet's enrollment profile then Fleet will report the "MDM turned off" state to Intune and the device will be automatically marked as non-compliant on Entra (even if it's passing all Fleet policies).

## Migration

> If you're migrating your macOS hosts from your current MDM solution to Fleet, you can switch to Fleet for conditional access with Entra before your MDM migration. This means you can also use a different MDM solution with Fleet for conditional access. 

Author: ```suggestion
If you're migrating your macOS hosts from your current MDM solution to Fleet, you can switch to Fleet for conditional access with Entra before your MDM migration. This means you can also continue to use your current MDM solution while using Fleet for conditional access. 
```
